spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binding/StreamListenerResultAdapter.java

```diff
@@ -41,6 +43,6 @@
 	 * @param streamListenerResult the result of invoking the method.
 	 * @param bindingTarget the binding target.
 	 */
-	void adapt(R streamListenerResult, B bindingTarget);
+	Closeable adapt(R streamListenerResult, B bindingTarget);
```

Comment:
@artembilan agree with the fact that this is a breaking change in 1.3. However, we discussed this with @mbogoevici before and came to the conclusions above - i.e. the `StreamListenerResultAdapter` implementations are internal exclusively at this point.